sources/Outputs/include/Network.h

```diff
@@ -0,0 +1,80 @@
+//
+// Copyright (c) 2022, RTE (http://www.rte-france.com)
+// See AUTHORS.txt
+// All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at http://mozilla.org/MPL/2.0/.
+// SPDX-License-Identifier: MPL-2.0
+//
+
+/**
+ * @file  Network.h
+ *
+ * @brief Dynaflow launcher Network file writer header file
+ *
+ */
+
+#pragma once
+
+#include "Configuration.h"
+#include "DynModelDefinitionAlgorithm.h"
```

Comment:
we do need DynModelDefinitionAlgorithm, DynamicDataBaseAlgorithm, LineDefinitionAlgorithm, ShuntDefinitionAlgorithm included here? I do not think so..